libkbfs/keybase_daemon_rpc.go

```diff
@@ -51,6 +53,11 @@ func NewKeybaseDaemonRPC(config Config, kbCtx Context, log logger.Logger, debug
 	conn := NewSharedKeybaseConnection(kbCtx, config, k)
 	k.fillClients(conn.GetClient())
 	k.shutdownFn = conn.Shutdown
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
May add a comment below in `newKeybaseDaemonRPCWithClient` that we don't need to bother doing a keepalive loop in that case.